packages/toolpad-app/src/toolpad/propertyControls/SelectOptions.tsx

```diff
@@ -48,11 +49,18 @@ function SelectOptionsPropEditor({
   const handleOptionTextInput = React.useCallback(
     (event: React.KeyboardEvent<HTMLDivElement>) => {
       if (event.key === 'Enter') {
-        onChange([...value, (event.target as HTMLInputElement).value]);
+        const inputText = (event.target as HTMLInputElement).value;
```

Comment:
Would the correct event type above in this function not be `React.KeyboardEvent<HTMLDivElement>`? That way you shouldn't need to do these type castings.

Reply:
The typecasing would continue to happen regardless; the issue is with `React.KeyboardEvent`. If that type is removed, then forced typecasting will be needed on the `event.key` line - so the problem exists regardless

Reply:
ah ok, we can leave it like this then